README.md

```diff
@@ -1 +1,63 @@
-# angular-what-is-a-controller-readme
\ No newline at end of file
+# What is a Controller?
+
+## Overview
+
+We've talked about controllers before, and now is the time to go into them. Controllers make up a lot of the code we write in Angular so it's important that we learn how to use them correctly and efficiently!
+
+## Objectives
+
+- Describe the role of a Controller
+- Create a Controller
+- Assign the Controller to the module
+- Create an element for Controller scope
+
+## Controllers
+
+You might have noticed that we've mentioned the concept of Controllers quite a lot. These are the backbone for your Angular applications, providing the link between the view and the model.
+
+Controllers differ slightly depending on your architecture. If we're using MVC our controller does a lot more than if we're using MVVM. The nicknames for the controller in these situations are fat and thin controllers.
+
+### Business/Presentational Logic
+
+We'll be using the terms "business" and "presentational" logic quite a lot, so it's important that we understand what these are.
+
+Business logic is all the logic we have to do in order to manipulate our model. For instance, retrieving data from an API endpoint and updating it. If we go back to our coffee shop example, the business logic would be the barista making our coffee.
+
+Presentational logic is when we adapt our model to be displayed correctly in the view. This would be the barista adding our name to the cup as they hand us the drink.
+
+### MVC: Fat Controller  
```

Comment:
In this section, can you mention the upsides/downsides to Fat vs Skinny controllers? Why/when would you choose one over the other? Could you even provide an example of applications or instances where one would be preferred over the other? These terms will be easier for the reader to understand with more context around how they're used.